include/erlcloud_ecs.hrl

```diff
@@ -188,6 +193,7 @@
     memory :: undefined | pos_integer(),
     memory_reservation :: undefined | pos_integer(),
     mount_points :: undefined | [#ecs_mount_point{}],
+    placement_strategy :: undefined | [#ecs_placement_strategy{}],
```

Comment:
should this go after `name`?

Reply:
actually, this is left over from a road not taken; deleting instead